cli/src/pcluster/templates/imagebuilder_stack.py

```diff
@@ -168,6 +169,19 @@ def _add_resources(self):
         lambda_cleanup = self._add_lambda_cleanup(lambda_cleanup_policy_statements, build_tags_list)
         self._add_sns_topic(lambda_cleanup, build_tags_list)
 
+    def _get_image_tags(self):
+        tags = copy.deepcopy(self.config.image.tags) if self.config.image and self.config.image.tags else []
+        # TODO add tags for build log
+        tag_list = [
```

Comment:
This code is duplicated, we should have a common function for preparing cfn tags list

Reply:
you mean it is duplicate comparing to imagebuilder_stack.py module or cluster.py module?

Reply:
done